docs/src/main/asciidoc/security-cors.adoc

```diff
@@ -11,26 +11,28 @@ include::_attributes.adoc[]
 :keywords: cors,http
 :extensions: io.quarkus:quarkus-vertx-http
 
-Cross-origin resource sharing (CORS) is an HTTP-header based mechanism that allows a server to indicate any origins (domain, scheme, or port) other than its own from which a browser should permit loading resources.
+Cross-origin resource sharing (CORS) is an HTTP-header-based mechanism that allows a server to indicate any origins other than its own, from which a browser should permit loading resources.
+
+These origins consist of a single domain, scheme, and port.
+For the complete origin definition, see the link:https://datatracker.ietf.org/doc/html/rfc6454[Web Origin Concept] page.
```

Comment:
Good idea :-), but do we know for sure that RFC does not talk about some origins which makes sense in context of CORS ? I don't mind keeping this reference, but I'd only keep in the Rerefences for the curious users to follow if they'd like

Reply:
The RFC though is quite technical, but lets keep, I'd only suggest to keep the link in References but I'll let you decide :+1: